x-pack/plugins/security_solution/cypress/integration/alerts_detection_rules_custom.spec.ts

```diff
@@ -120,6 +121,7 @@ describe('Custom detection rules creation', () => {
 
   after(() => {
     esArchiverUnload('timeline');
+    deleteRule();
```

Comment:
Are these two operations totally independent? I assume so but wanted to call out the discrepancy in ordering.

Reply:
In this specific case, does not matter the other, but you are right, in general, it should not be like that, good catch! I need to update this 😊